solr/core/src/java/org/apache/solr/cluster/placement/PlacementPluginFactory.java

```diff
@@ -17,15 +17,39 @@
 
 package org.apache.solr.cluster.placement;
 
+import org.apache.solr.api.ConfigurablePlugin;
+
 /**
- * Factory implemented by client code and configured in {@code solr.xml} allowing the creation of instances of
+ * Factory implemented by client code and configured in container plugins allowing the creation of instances of
```

Comment:
Can we add a reference to how "container plugins" are configured? Not obvious for me and I guess I'm likely not the only one...

Reply:
(referencing the Affinity class and its configuration would likely be good enough here)

Reply:
I'll add a reference to `ContainerPluginsApi#Edit` - JSON formats and the location of the config in `/clusterprops.json` are an implementation detail. Also, container plugin configuration is/will be described in the RefGuide (SOLR-15022).